tmt/convert.py

```diff
@@ -46,6 +46,116 @@ def repr_str(dumper, data):
 #  Convert
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+def read_manual(plan_id, case_id, disabled):
+    """
+    Reads metadata of manual test cases from Nitrate
+    """
+    try:
+        import nitrate
```

Comment:
We should probably use/share the `import_nitrate()` function from `export.py` and catch possible `gssapi` exceptions as it is done there.

Reply:
fixed in 440e5c1
Added returning of the module to the import_nitrate() method so it can be used outside of its scope.